src/transform/HeaderTransform.css

```diff
@@ -0,0 +1,63 @@
+.pagelib_edit_section_link {
+  float: right;
+}
+
+/* todo: standardize on RTL selectors throughout the codebase. In many places we use .content-rtl. */
+html[dir='rtl'] .pagelib_edit_section_link,
+.content-rtl .pagelib_edit_section_link {
+  float: left;
+}
+
+.pagelib_pronunciation svg,
+.pagelib_header_description svg {
+  vertical-align: middle;
+  fill: currentColor;
+}
+
+html[dir='rtl'] .pagelib_edit_section_link,
+.content-rtl .pagelib_edit_section_link,
+html[dir='rtl'] .pagelib_pronunciation svg,
+.content-rtl .pagelib_pronunciation svg,
+html[dir='rtl'] .pagelib_header_description svg,
+.content-rtl .pagelib_header_description svg {
+  transform: scaleX(-1);
+}
+
+.pagelib_header_description {
+  font-size: 90%;
+}
+
+.pagelib_pronunciation {
+  color: inherit;
+}
+
+.pagelib_header_description a {
+  font-style: italic;
+}
+
+.pagelib_divider {
+  width: 56px;
+  border-top: 1px solid currentColor !important;
+  margin-top: 12px;
+  margin-bottom: 16px;
+}
+
+.pagelib_edit_section_link,
+.pagelib_header_description,
+.pagelib_divider {
+  opacity: 0.7;
+}
+
+.pagelib_theme_dark .pagelib_edit_section_link,
+.pagelib_theme_black .pagelib_edit_section_link,
+.pagelib_theme_dark .pagelib_header_description,
+.pagelib_theme_black .pagelib_header_description,
+.pagelib_theme_dark .pagelib_divider,
+.pagelib_theme_black .pagelib_divider {
+  opacity: 0.54;
+}
+
+.pagelib_theme_dark .pagelib_edit_section_link,
+.pagelib_theme_black .pagelib_edit_section_link {
+  -webkit-filter: invert(100%);
```

Comment:
I think this is not needed. The EditTransform is responsible for theming its edit pencils.

Reply:
Same for lines 51 and 52.

Reply:
I think I got this one corrected now.